go/storage/api/grpc.go

```diff
@@ -21,54 +21,45 @@ var (
 	ServiceName = cmnGrpc.NewServiceName("Storage")
 
 	// MethodSyncGet is the SyncGet method.
-	MethodSyncGet = ServiceName.NewMethod("SyncGet", GetRequest{})
-	// MethodSyncGetPrefixes is the SyncGetPrefixes method.
-	MethodSyncGetPrefixes = ServiceName.NewMethod("SyncGetPrefixes", GetPrefixesRequest{})
-	// MethodSyncIterate is the SyncIterate method.
-	MethodSyncIterate = ServiceName.NewMethod("SyncIterate", IterateRequest{})
-	// MethodApply is the Apply method.
-	MethodApply = ServiceName.NewMethod("Apply", ApplyRequest{}).
+	MethodSyncGet = ServiceName.NewMethod("SyncGet", GetRequest{}).
 			WithNamespaceExtractor(func(ctx context.Context, req interface{}) (common.Namespace, error) {
-			r, ok := req.(*ApplyRequest)
+			r, ok := req.(*GetRequest)
 			if !ok {
 				return common.Namespace{}, errInvalidRequestType
 			}
-			return r.Namespace, nil
+			return r.Tree.Root.Namespace, nil
 		}).
 		WithAccessControl(func(ctx context.Context, req interface{}) (bool, error) {
```

Comment:
```
func(ctx context.Context, req interface{}) (bool, error) {
  return true, nil
}
```
could be extracted into common/grpc package, so one could then do something like:
`.WithAccessControl(cmnGrpc.AccessControlAlways)` when there's no special logic for deciding on access control.